recipes/class_loader/build.sh

```diff
@@ -0,0 +1,30 @@
+mkdir build
+cd build
+
+# turn this into a catkin workspace by adding the catkin token if it doesn't
+# exist already
+if [ ! -f $PREFIX/.catkin ]; then
+    touch $PREFIX/.catkin
+fi
+
+# necessary for correctly linking SIP files (from python_qt_bindings)
+export LINK=$CXX
+
+export ROS_PYTHON_VERSION=`${PYTHON} -c "import sys; print('%i.%i' % (sys.version_info[0:2]))"`
```

Comment:
Is this only needed during the build or would this also be useful as an activation script?

Reply:
I think it's safer to set it like this. It might be set in the ros_environment package.